kafka/kontena.yml

```diff
@@ -0,0 +1,81 @@
+---
+stack: kontena/kafka-cluster
+version: 1.0.0
+description: Kafka cluster based on Harvard Hypermap
+expose: kafka
+variables:
+  num_zookeeper_instances:
+    type: integer
+    min: 1
+    default: 3
+    from:
+      env: NUM_ZOOKEEPER_INSTANCES
+      prompt: number of instances of Zookeeper?
+  num_kafka_instances:
+    type: integer
+    min: 1
+    default: 3
+    from:
+      env: NUM_KAFKA_INSTANCES
+      prompt: number of instances of Kafka?
+  expose_kafka_port:
+    type: boolean
+    default: false
+    from:
+      env: EXPOSE_KAFKA_PORT
+      prompt: expose Kafka port to host?
+
+services:
+  zookeeper:
+    image: harvardcga/zookeeper:3.4
+    stateful: true
+    volumes:
+      - zookeeper-data:/var/lib/zookeeper
+    instances: ${num_zookeeper_instances}
+    deploy:
+      wait_for_port: 2181
+      min_health: 0.5
+    environment:
+      # {% assign zookeeper_servers = "" %}
+      # {% for i in (1..num_zookeeper_instances) %}
+      # {%   capture server %}server.{{ i }}=zookeeper-{{ i }}.{{ STACK }}.{{ GRID }}.kontena.local:2888:3888 {% endcapture %}
+      # {%   assign zookeeper_servers = zookeeper_servers | append: server %}
+      # {% endfor %}
+      ZOO_SERVERS: {{ zookeeper_servers }}
```

Comment:
This is kind of an unfortunate thing, that we need to tell Zookeeper explicitly about the other instances.  It works quite nicely at startup w/ the num_instances var, but I can't image this going so well in a `k service scale` scenario.